app/assets/stylesheets/logixRelease.scss

```diff
@@ -336,7 +336,7 @@ a.btn.btn-outline-danger {
 }
 
 .fa-post {
-  font: normal normal normal 14px/1 FontAwesome;
+  font: normal normal normal 14px/1 sans-serif;
```

Comment:
Curios why?

Reply:
I tried to change the font but there were alignment issues. So changed it like this